### PR TITLE
Clan actions for new clan page

### PR DESCRIPTION
**Pull request by Paul-Vasylenko**

### CODE REVIEW

**Comment by Paul-Vasylenko** [comment]
[[Clan] Add make admin](https://trello.com/c/IayvShEw/245-clan-add-make-admin)

**Comment by Paul-Vasylenko** [comment]
[Admin of the clan should be able to delete clan members and admins](https://trello.com/c/0HqVG074/259-admin-of-the-clan-should-be-able-to-delete-clan-members-and-admins)

File: client/src/components/pages/clans/clan/components/clan-actions/index.tsx [hunk 1]
```diff
@@ -17,9 +24,14 @@ const ClanActions: FC<IClanActionsProps> = ({ visitor, handleJoin, handleLeave,
...
-			<Button className={clsx(ButtonClasses.red, ButtonClasses.filled)} onClick={handleEdit}>
-				Edit
-			</Button>
+			<>
+				<Button className={clsx(ButtonClasses.red, ButtonClasses.filled)} onClick={handleEdit}>
+					Edit
+				</Button>
+				<Button className={clsx(ButtonClasses.red, ButtonClasses.filled)} onClick={handleDelete}>
+					Delete
+				</Button>
```
Comment (INTJT) [inline client/src/components/pages/clans/clan/components/clan-actions/index.tsx:33]: Delete clan button already exists in edit clan modal. Remove it from modal / clan-actions component.

File: server/src/api/profile-clan/profile-clan-api.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	const { profileClan: profileClanService } = services;
+
+	const router = appRouter();
+
+	router.put(ProfileClanApiPath.TO_ONE_BY_ID, async (req, res, next) => {
+		profileClanService
+			.updateRole(req.params.id,req.body.role)
+			.then((data)=>res.send(data))
+			.catch(next);
+	});
```
Comment (INTJT) [inline server/src/api/profile-clan/profile-clan-api.ts:15]: Only admin of own clan can give to other members the admin role. Make it by middleware / service

**Review by INTJT — CHANGES_REQUESTED** [review]

**Review by INTJT — APPROVED** [review]

**Pull request merged**